hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/auditing.md

```diff
@@ -221,22 +222,23 @@ https://audit.example.org/hadoop/1/op_rename/3c0d9b7e-2a63-43d9-a220-3c574d768ef
 Here are the fields which may be found in a request.
 If any of the field values were `null`, the field is omitted.
 
-| Name | Meaning | Example |
-|------|---------|---------|
-| `cm` | Command | `S3GuardTool$BucketInfo` |
-| `fs` | FileSystem ID | `af5943a9-b6f6-4eec-9c58-008982fc492a` |
-| `id` | Span ID | `3c0d9b7e-2a63-43d9-a220-3c574d768ef3-3` |
-| `ji` | Job ID  (S3A committer)| `(Generated by query engine)` |
-| `op` | Filesystem API call | `op_rename` |
-| `p1` | Path 1 of operation | `s3a://alice-london/path1` |
-| `p2` | Path 2 of operation | `s3a://alice-london/path2` |
-| `pr` | Principal | `alice` |
-| `ps` | Unique process UUID | `235865a0-d399-4696-9978-64568db1b51c` |
-| `rg` | GET request range | `100-200` |
-| `ta` | Task Attempt ID (S3A committer) | |
-| `t0` | Thread 0: thread span was created in | `100` |
-| `t1` | Thread 1: thread this operation was executed in | `200` |
-| `ts` | Timestamp (UTC epoch millis) | `1617116985923` |
+| Name | Meaning                                                                                              | Example |
```

Comment:
don't do the ide reformat as it changes the width of the table every time. just add the new row